auto_submit/lib/service/validation_service.dart

```diff
@@ -128,40 +143,148 @@ class ValidationService {
     for (ValidationResult result in results) {
       if (!result.result && result.action == Action.REMOVE_LABEL) {
         final String commmentMessage = result.message.isEmpty ? 'Validations Fail.' : result.message;
-        await gitHubService.createComment(slug, prNumber, commmentMessage);
-        await gitHubService.removeLabel(slug, prNumber, Config.kAutosubmitLabel);
-        log.info('auto label is removed for ${slug.fullName}, pr: $prNumber, due to $commmentMessage');
+
+        String message = 'auto label is removed for ${slug.fullName}, pr: $prNumber, due to $commmentMessage';
+
+        log.info(message);
```

Comment:
Move the log after `removeLabelAndComment` to make sure it's logged only when it does happen.